format/format.sh

```diff
@@ -63,28 +67,44 @@ case "$mode" in
  *) echo >&2 "unknown mode $mode";;
 esac
 
-files=$(git ls-files '*/BUILD.bazel' '*.bzl' '*.BUILD' 'WORKSPACE' '*.bazel')
+if [ "$#" -eq 0 ]; then
+  files=$(git ls-files '*/BUILD.bazel' '*.bzl' '*.BUILD' 'WORKSPACE' '*.bazel')
+else
+  files=$(find "$@" -name '*/BUILD.bazel' -or -name '*.bzl' -or -name '*.BUILD' -or -name 'WORKSPACE' -or -name '*.bazel')
```

Comment:
Does this need to respect `.gitignore`?

Reply:
Oh, this just filters the files from the cli args?

Reply:
correct